parsl/monitoring/db_manager.py

```diff
@@ -660,7 +662,9 @@ def _insert(self, table: str, messages: List[MonitoringMessage]) -> None:
                     done = True
                 except sa.exc.OperationalError as e:
                     # hoping that this is a database locked error during _update, not some other problem
-                    logger.warning("Got a database OperationalError. Ignoring and retrying on the assumption that it is recoverable: {}".format(e))
+                    logger.warning(
+                        f"Got a database OperationalError."
```

Comment:
Missing space at the end of this string - putting two `""`-strings next to each other doesn't put a space between them, but the original message did have a space.